packages/gestalt/src/Tabs.js

```diff
@@ -53,10 +55,16 @@ export default class Tabs extends React.Component<Props, State> {
   handleTabMouseLeave = () => this.setState({ hoveredTabIndex: undefined });
 
   render() {
-    const { tabs, activeTabIndex } = this.props;
+    const { tabs, activeTabIndex, wrap } = this.props;
     const { focusedTabIndex, hoveredTabIndex } = this.state;
     return (
-      <div className={styles.Tabs} role="tablist">
+      <div
+        className={classnames(
+          styles.Tabs,
+          wrap !== undefined && styles.TabsWrap
```

Comment:
This logic is wrong, imagine I set `wrap=false`, in that case `false !== undefined` evaluates to `true` so `true && styles.TabsWrap` would set that CSS class on this component when in fact we didn't want it because we passed `false`.